src/main/resources/static/openapi/EducationAndWorkPlanAPI.yml

```diff
@@ -119,7 +119,7 @@ paths:
       summary: Updates a Goal
       description: Updates a single Goal and its list of Steps (including their sequential order if relevant). Note that the whole of a Goal's data should be provided, otherwise the absence of any data (such as a Step) will result in it being removed.
       tags:
-        - Action Plan - Single Goal
+        - Action Plan Goals
```

Comment:
Minor - how about just `Goals`, rather than `Action Plan Goals` (which doesn't read quite right to me)? Or perhaps `Action Plan - Goals`?

Also, should this be Goal singular or plural here?

Reply:
I'm happy to change it to just `Goals`; but I think it should be plural. I think even if we hadn't have changed the create goal to create goals I still think it would be plural. The `tags` element is about grouping common/related APIs, and I think they are all "about" goals. I think that is also borne by the fact that our URL path is goals (plural)
I'll change it not and see what we think think ....

Reply:
I think it looks good now 👍 
![image](https://github.com/ministryofjustice/hmpps-education-and-work-plan-api/assets/135589132/0a3b2f05-9897-4b7b-b1df-2079c909921f)

Reply:
lol - I did consider changing them, but decided better of it! 🤣 
One for another day